portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -14,19 +14,49 @@
 
 package com.google.sps.servlets;
 
+import com.google.gson.Gson;
 import java.io.IOException;
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import java.util.*;
 
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
+    private ArrayList<String> list;
+
+    @Override
+    public void init(){
+        list = new ArrayList<String>();
+            list.add("Brandon");
+            list.add("Jones");
+            list.add("18"); 
+    }   
+
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
     response.setContentType("text/html;");
-    response.getWriter().println("<h1>Hello world!</h1>");
+    String Json = convertToJson(list);
+    //response.getWriter().println("<h1>Hello Brandon!</h1>");
+    response.setContentType("application/json;");
+    response.getWriter().println(Json);
+
+  }
+  private String convertToJsonUsingGson(ArrayList<String> list) {
+    Gson gson = new Gson();
+    String json = gson.toJson(list);
+    return json;
```

Comment:
Your hand written json string below is a map, not a list.

To make the output of this function consistent with the hand written json, build the hashmap

HashMap<String, String> map = new HashMap<String,String>();
map.put("firstName", "Brandon");
...
String json = gson.ToJson(map);